php-nginx/entrypoint.sh

```diff
@@ -138,4 +134,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
+# Remove the loose php-cli.ini
+rm ${PHP_DIR}/lib/php-cli.ini
```

Comment:
FYI I tested running composer with suhosin and `display_errors=STDERR`, but it didn't display anything.